src/Dompdf.php

```diff
@@ -778,7 +793,7 @@ public function render()
                     $canvas->register_string_subset($style->font_family, $chars);
 
                     // the raw text of the content property
-                    $canvas->register_string_subset($style->font_family, $style->content);
+                    $this->getCanvas()->register_string_subset($style->font_family, $style->content);
```

Comment:
How come you're not using the $canvas variable already populated with the current canvas?